test/check-application

```diff
@@ -1257,6 +1258,12 @@ class TestApplication(testlib.MachineCase):
         b.click('.pf-c-modal-box button:contains(Restore)')
         b.wait(lambda: self.getContainerAttr("swamped-crate", "State") in 'Running')
 
+        # A new MAC address should have been generated
+        _, _, version = self.execute(False, "podman -v").split(' ')
+        # Fixed in podman 4.4.0 https://github.com/containers/podman/issues/16666
+        if int(version[0]) > 3 and int(version[2]) > 3:
```

Comment:
This is confusing syntax :) That `.` is `version[1]` maybe you can split it on `.`.

Reply:
Looked at it again and re-worked it so it's more logical.